serviceAccessConfig/usr/sbin/serviceAccessConfig

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/python
+
+#  Copyright (C) 2016 SUSE LLC, Robert Schweikert <rjschwei@suse.com>
+#  All rights reserved.
+#
+#  This file is part of serviceAccessConfig
+#
+#  This program is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+"""Command serviceAccessConfig
+
+Usage:
+    serviceAccessConfig  -h | --help
+    serviceAccessConfig [ --config --ipdata --log ]
```

Comment:
I don't think that does what you think it does [1]... you're going to end up with boolean options.

I think you want something like
```
serviceAccessConfig [--config=<config-file-name>] [--ipdata=<ip-data-file-name>] [--log=<log-file-name>]
```

[1] http://try.docopt.org/?doc=Command+serviceAccessConfig%0D%0AUsage%3A%0D%0A++++serviceAccessConfig++-h+|+--help%0D%0A++++serviceAccessConfig+[+--config+--ipdata+--log+]%0D%0A++++serviceAccessConfig+--version%0D%0AOptions%3A%0D%0A++++--config%0D%0A++++++++The+configuration+file+for+serviceAccessConfig%0D%0A++++-h+--help%0D%0A++++++++Show+help%0D%0A++++--ipdata%0D%0A++++++++The+source+data+file+to+monitor+from+which+the+CIDR+blocks+will%0D%0A++++++++be+extracted+to+generate+the+access+data+for+the+configured+service%28s%29%0D%0A++++--log%0D%0A++++++++The+log+file+to+be+written&argv=--config%3D%22test.cfg%22